source/iopipe/json/serialize.d

```diff
@@ -2545,3 +2555,13 @@ unittest
 
 	assert(iota(10).map!"a+1".serialize == "[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]");
 }
+
+unittest
+{
+    static struct S
+    {
+        int x;
+    }
+    auto tok = `{"x": 1}`.jsonTokenizer;
+    assert(DefaultDeserializationPolicy!false().deserializeObject!S(tok) == S(1));
```

Comment:
Like, this call can just be `tok.deserialize!S(DefaultDeserializationPolicy!false())`

Note that `deserializeObject` is meant to be a policy-driven building block, not for the user. In this way, you are essentially overriding the policy decision that the type given is deserialized via a JSON object.